### PR TITLE
Add SUB to prices_bnb_tokens.sql

### PR DESCRIPTION
**Pull request by 0xfraan**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

**Comment by Hosuke** [comment]
> **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you all sign our [Contributor License Agreement](https://github.com/duneanalytics/spellbook/blob/main/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> 
> I have read the CLA Document and I hereby sign the CLA
> 
> **1** out of **2** committers have signed the CLA.✅ @Hosuke❌ @0xfraanYou can retrigger this bot by commenting **recheck** in this Pull Request

Hi @0xfraan, please sign the CLA so we can continue this PR.

**Comment by 0xfraan** [comment]
I have read the CLA Document and I hereby sign the CLA

File: models/prices/bnb/prices_bnb_tokens.sql [hunk 1]
```diff
@@ -204,7 +204,8 @@ FROM
     ('joyg-kismet-joy-gold', 'bnb', 'JOYG', 0xe63c37e50a045c247b5003e24ba066f1cc3f0ff3, 8),
     ('gft-galaxy-finance', 'bnb', 'GFT', 0xe77932b1216125848e82c3967e75698362168f99, 18),
     ('snbnb-synclub-staked-bnb', 'bnb', 'SnBNB', 0xb0b84d294e0c75a6abe60171b70edeb2efd14a1b, 18),
-    ('gmt-stepn', 'bnb', 'GMT', 0x3019BF2a2eF8040C242C9a4c5c4BD4C81678b2A1, 8)
+    ('gmt-stepn', 'bnb', 'GMT', 0x3019BF2a2eF8040C242C9a4c5c4BD4C81678b2A1, 8),
+    ('sub-subme', 'bnb', 'SUB', 0x47bB3D817e627D196ae74b89AfA7928C2Bb159A9, 18)
```
Comment (Hosuke) [inline models/prices/bnb/prices_bnb_tokens.sql:208]: API: https://api.coinpaprika.com/v1/coins/sub-subme

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @0xfraan

File: models/prices/bnb/prices_bnb_tokens.sql (hunk 1 above)
Comment (Hosuke) [reply models/prices/bnb/prices_bnb_tokens.sql:208]: SCAN: https://bscscan.com/token/0x47bB3D817e627D196ae74b89AfA7928C2Bb159A9

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**